### PR TITLE
fixed some typo

### PR DESCRIPTION
**Pull request by sinAshish**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](http://bugs.python.org/) (b.p.o) to verify you have signed the CLA. This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://cpython-devguide.readthedocs.io/pullrequest.html#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: experts.rst [hunk 1]
```diff
@@ -3,11 +3,11 @@
 Experts Index
 =============
 
-This document has tables that list Python Modules, Tools, Platforms and
-Interest Areas and names for each item that indicate a maintainer or an
+This document has tables that lists Python Modules, Tools, Platforms and
+Interest Areas and names for each item that indicates a maintainer or an
```
Comment (DimitrisJim) [inline experts.rst]: The original versions are correct.

File: experts.rst [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
 expert in the field.  This list is intended to be used by issue submitters,
 issue triage people, and other issue participants to find people to add to
-the nosy list or to contact directly by email for help and decisions on
+the nosy list or to contact them directly by email for help and decisions on
```
Comment (DimitrisJim) [inline experts.rst]: I think both are correct (i.e this boils down to preference.)

File: experts.rst (hunk 1 above)
Comment (terryjreedy) [inline experts.rst]: I agree.

File: experts.rst (hunk 2 above)
Comment (terryjreedy) [inline experts.rst]: I prefer the original.

File: experts.rst (hunk 1 above)
Comment (sinAshish) [inline experts.rst]: ok...thanx for reviewing so quickly

**Pull request closed without merging**